### PR TITLE
Issue #1669: Adding extension method ConfigureRouteOptions.

### PR DESCRIPTION
**Pull request by sornaks**
Linking https://github.com/aspnet/Mvc/issues/1669

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@sornaks__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft Open Technologies, Inc. and real humans are currently evaluating your PR.</p>

TTYL, MSOTBOT;

File: src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Routing;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    /// <summary>
+    /// Contains extension methods to IServiceCollection.
```
Comment (rynowak) [inline src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs]: `<see >`

File: src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    /// </summary>
+    public static class ServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Configures a set of <see cref="RouteOptions"/> for the application.
+        /// </summary>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="setupAction">The <see cref="RouteOptions"/> which need to be configured.</param>
```
Comment (rynowak) [inline src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs]: This isn't accurate

**Comment by rynowak** [comment]
:shipit: after fixing minor comments

**Comment by sornaks** [comment]
Fixed minor comments and checked in. - https://github.com/aspnet/Routing/commit/67dcdbf8a1f4c1f45f8ce1a245e726bb14fb7ac9

**Pull request closed without merging**